CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Set the default port to 8150.
```

Comment:
@kopiczko isn't there a consumer of this port? do we need to adapt somewhere else?

Reply:
There isn't any service bound to this port known to me. If there is. It seems to work as I was testing it.

Reply:
`net-exporter` service is backed by this port. As far as I see, helm templating will update it as well. However, that service has `giantswarm.io/monitoring: "true"` label and Prometheus scrapes metrics via this port. I checked whether we need to update somewhere for this port and found this line https://github.com/giantswarm/prometheus-meta-operator/blob/master/files/templates/scrapeconfigs/additional-scrape-configs.template.yaml#L451

Could you please check Prometheus config&service discovery after updating this port?

Reply:
I tested that multiple times.